### ISSUE TITLE
[Programmatic Access - Foundry Local - Models]: 'Sort by' label is not associated with its combo field under 'Models' page.

### ISSUE THREAD

**Issue by chandini1864** [microsoft/foundry-local#1005]
### Describe the issue

[A11y Agency plugin ](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fvigilant-adventure-v9qpqwn.pages.github.io%2fall%2f%23plugins%2fa11y?q%3Da11y&data=05%7c02%7cv-cmadda%40microsoft.com%7cc809251b79d74200ecc908decb9453c2%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639172035135569684%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=5v%2BYGmPOuGLwcQ4QzpTYDdOQmR2PxYEy1h5Z1mZ2ykI%3D&reserved=0)is available to help with accessibility bug remediation.
The required tag has already been added: agency:agent=a11y:a11y-agent@company. Assign the bug to Agency to use it.
Please try it and share feedback on whether the remediation guidance was useful at [CAIA11yAgency@microsoft.com](mailto:CAIA11yAgency@microsoft.com).

**User Exprience:**
Screen reader users may not hear the label when they focus a form field, preventing them from readily knowing the field’s purpose. They must explore surrounding text to find context.

Note:  User credentials should NOT be included in the bug.

**Actual Result:**
'Sort by' label is not programmatically associated with its last modified combo field under 'Models' page.

**Note:**
Same issue for Model Family and Acceleration como fields under Models page. 

**Expected Result:**
'Sort by' label should be programmatically associated with its last modified combo field under 'Models' page.

**How to Fix:**
Use <label for="fieldId"> matching the input’s id. When a native label cannot be used, you can use aria-labelledby to reference the visible label text. 

### To reproduce

1. Open the URL "https://www.foundrylocal.ai/" and login with valid credentials.
2. Foundry local homepage opens. Select 'Models' from header.
3. Navigate to 'Sort by' combo field under 'Browse Foundry Models' section.
4. Verify whether sort by label is associated with its combo field or not.

### Platform and architecture

Windows X64

### OS Version

Windows 25H2 (OS Build 26200.8893)

### Installation type

Released package/binary

### Foundry Local version

https://www.foundrylocal.ai/

### API or surface area

CLI

### Hardware acceleration/backend

Other / Unknown

<img width="1074" height="581" alt="Image" src="https://github.com/user-attachments/assets/072e93da-4779-4fd9-9375-267f983d89f6" />

**Comment by chandini1864** [comment]
#A11yMAS;#A11ySev2;#HCL;#Accessibility;#Benchmark;#GH-HCL-FoundryLocal-Aug2026;#MAS1.3.1;#Win11-Edge(Chromium);#agency:agent=a11y:a11y-agent@company;#a11ydb-label-not-associated;

**Comment by prathikr** [comment]
@chandini1864 this issue is fixed. Re-assigning this issue back to you to re-run accessibility analysis.
